script/test.sh

```diff
@@ -128,17 +138,17 @@ curl -XPOST localhost:9000/graphs/createServiceColumn -H 'Content-Type: Applicat
 }
 ' 
 # add props on Vertex
```

Comment:
changed api name from addVertexProps to addServiceColumnProp. checkout https://github.com/daumkakao/s2graph/blob/master/conf/routes#L66